recipes/WSJ0Mix/separation/hparams/sepformer-conformerintra.yaml

```diff
@@ -0,0 +1,170 @@
+# ################################
+# Model: SepFormer with Conformer block in the intra processing block
+# Dataset : WSJ0-2mix and WSJ0-3mix
+# ################################
+#
+# Basic parameters
+# Seed needs to be set at top of yaml, before objects with parameters are made
+#
+seed: 1234
+__set_seed: !apply:torch.manual_seed [!ref <seed>]
+
+# Data params
+
+# e.g. '/yourpath/wsj0-mix/2speakers'
+# end with 2speakers for wsj0-2mix or 3speakers for wsj0-3mix
+data_folder: !PLACEHOLDER
+
+# the path for wsj0/si_tr_s/ folder -- only needed if dynamic mixing is used
+# e.g. /yourpath/wsj0-processed/si_tr_s/
+# you need to convert the original wsj0 to 8k
+# you can do this conversion with the script ../meta/preprocess_dynamic_mixing.py
+base_folder_dm: /yourpath/wsj0-processed/si_tr_s/
```

Comment:
I like the example in the comments
`!PLACEHOLDER` will serve well here

Reply:
@anautsch , I prefer not having the `!PLACEHOLDER` here, as I think it's better not to have code crash in case where we do not use dynamic mixing.

Reply:
good to know ! :D